Lib/sqlite3/dbapi2.py

```diff
@@ -86,7 +86,7 @@ def convert_timestamp(val):
 
 # bpo-24464: enable_shared_cache was deprecated in Python 3.10.  It's
 # scheduled for removal in Python 3.12.
-_old_enable_shared_cache = enable_shared_cache
+from _sqlite3 import enable_shared_cache as _old_enable_shared_cache
```

Comment:
Imports are generally all at the top of the module in the stdlib

Reply:
Normally, yes, but in this case I prefer to keep it together with the override. Actually I could put it inside the override as an improvement.